features/staging_commands.feature

```diff
@@ -542,3 +542,36 @@ Feature: Enable command behaviour when attached to an UA staging subscription
            | release | fips-apt-source                                                        |
            | xenial  | https://esm.staging.ubuntu.com/fips-updates/ubuntu xenial-updates/main |
            | bionic  | https://esm.staging.ubuntu.com/fips-updates/ubuntu bionic-updates/main |
+
+
+    @series.lts
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Commands when cloud init didn't run properly
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I delete the file `/run/cloud-init/instance-data.json`
+        And I attach `contract_token_staging` with sudo
+        Then stdout matches regexp:
+        """
+        Warning: cloud-init is present but has not detected the cloud. Assuming
+        not on cloud. If you are on a cloud, you may experience unreliable behavior.
+        """
+        When I run `ua status` with sudo
+        Then stdout matches regexp:
+        """
+        Warning: cloud-init is present but has not detected the cloud. Assuming
+        not on cloud. If you are on a cloud, you may experience unreliable behavior.
+        """
+        When I run `ua disable livepatch` with sudo
+        And I run `ua enable fips` `with sudo` and stdin `y`
```

Comment:
I think we should only add one more step here to guarantee that fips was enabled in the system, like checking `ua status` for example